warehouse/query-core/src/main/java/datawave/query/jexl/visitors/BaseIndexExpansionVisitor.java

```diff
@@ -182,14 +180,31 @@ protected void rebuildFutureJexlNodes() {
 
             rebuildFutureJexlNode(futureJexlNode);
 
-            JexlNode newNode = futureJexlNode.getRebuiltNode();
+            if (futureJexlNode.jjtGetParent() == null) {
```

Comment:
a future jexl node can never be the root of the tree?

Reply:
That is correct. They were previously embedded in the node tree but not anymore because that was dumb.